event-store-mem/README.md

```diff
@@ -0,0 +1,57 @@
+event-store-mem: Event Store that stores events in memory
+=========================================================
+
+Level: Beginner  
+Technologies: JavaEE  
+Summary: Example of Event Store provider
+Target Product: Keycloak  
+Source: <https://github.com/keycloak/keycloak-quickstarts>
+
+What is it?
+-----------
+
+This example shows how to implement and deploy an event store provider that stores events in memory.
+You can extend the server's functionality in this way by implementing your own provider. For more information see our server developer guide.
+
+After the provider is deployed and enabled in Keycloak, log out and log in back. Go to [Login Events](http://localhost:8180/auth/admin/master/console/#/realms/master/events) tab in the admin console where you can see login events showed from the Event store provider.
```

Comment:
This needs to detail how to enable the event provider in Keycloak manually. https://github.com/keycloak/keycloak/blob/master/examples/providers/event-store-mem/README.md